firebase-config/ktx/src/main/kotlin/com/google/firebase/remoteconfig/ktx/RemoteConfig.kt

```diff
@@ -45,6 +53,29 @@ fun remoteConfigSettings(
   return builder.build()
 }
 
+/**
+ * Starts listening for config updates from the Remote Config backend and emits [ConfigUpdate]s via
+ * a [Flow]. See [FirebaseRemoteConfig.addOnConfigUpdateListener] for more information.
+ *
+ * - When the returned flow starts being collected, an [ConfigUpdateListener] will be attached.
+ * - When the flow completes, the listener will be removed. If there are no attached listeners,
+ *   the connection to the Remote Config backend will be closed.
+ */
+fun FirebaseRemoteConfig.configUpdates(): Flow<ConfigUpdate> {
+  return callbackFlow {
+    val registration = Firebase.remoteConfig.addOnConfigUpdateListener(object: ConfigUpdateListener {
+      override fun onUpdate(configUpdate: ConfigUpdate) {
+        trySendBlocking(configUpdate)
```

Comment:
Ideally we'd call `trySendBlocking` in a background thread to avoid blocking the main thread. Does the Remote Config SDK have a background executor that can be used here?

For instance, in RTDB we leveraged the existing scheduler: https://github.com/firebase/firebase-android-sdk/blob/c062de8e689b037beed7f5f3a0faafe2bf9fea90/firebase-database/ktx/src/main/kotlin/com/google/firebase/database/ktx/Database.kt#L85

Reply:
Ah, thanks. I wasn't sure if `trySendBlocking` would be called in the same thread as the callback itself (which is called with a background executor. I updated `FirebaseRemoteConfig` to expose its executor and use that here. LMK if that's what you had in mind.

Reply:
I think that should work :)